azure-pipelines.yml

```diff
@@ -41,6 +41,10 @@ jobs:
         image: [linux, windows, macOs]
       dev: null
       package_description: null
+    before:
+    - script: '.tox\py27\Scripts\python src\tox\interpreters\windows\pep514.py'
```

Comment:
this gets imported as a side-effect of the tests -- do we need this separate one?

Reply:
I'm debugging why we pick up the Conda one rather than the provisioned one, so I want to see the stdout of this. Note with conda things passed though, but want to ensure not picking up conda from registry as valid for py27...